packages/auth/tests/session.spec.ts

```diff
@@ -528,21 +570,21 @@ describe('Wallet integration', function () {
         }
       })
 
-      expect(session.authPromises.length).to.equal(1)
-      await Promise.all(session.authPromises.map((p) => p.promise))
+      expect((session as any)._allInitialAuthRequests.length).to.equal(networksWithApi.length)
```

Comment:
is this `as any` so you can pull out the private property? seems stinky to be testing an internal property of the system